### ISSUE TITLE
positioning constraints on turntable camera

### ISSUE THREAD

**Issue by raacampbell13** [vispy/vispy#1088]
Are there any reasons for the positioning constraints on the turntable camera? e.g. I can only specify positions along one half of the circle in elevation (+90 to -90). Similar things happen in azimuth. This makes viewing the plot from certain angles impossible.

**Comment by Eric89GXL** [comment]
From what I remember, the elevation is constrained but the azimuth is not (or azimuth is modulo'ed to -pi<->pi, or 0<->2pi, which should be functionally equivalent). This is done to help maintain proper orientation ("up" direction) for the user. What angles can't you get to? With a +90<->-90 elevation and full range of azimuths, you should be able to get to all angles, can you not?

While I could see some value in adding a `kwarg` for constraining the elevation values or not since it could make getting to certain viewing angles easier, it will also lead to strange behaviors. For example, once you have rotated 180 degrees in elevation, a rightward mouse movement will lead to a leftward rotation, and vice-versa. In my mind this potential confusion is not worth the slight gains in viewing flexibility, so I'm -1 on adding it.

Maybe what you really want for your use case is the arcball camera?

**Comment by raacampbell13** [comment]
Yes, the arcball is better but I couldn't see a way of programmatically setting the view direction. I'm trying to make movies so I need to increment azimuth and elevation from a loop.
